docs/ref/forms/api.txt

```diff
@@ -1045,6 +1045,20 @@ Methods of ``BoundField``
         >>> print(bound_form['subject'].value())
         hi
 
+.. method:: BoundField.widget_type()
```

Comment:
Needs a `.. versionadded: 3.1`

Reply:
Also it is a property, so change this line to:

```rst
.. attribute:: BoundField.widget_type
```